k8s/prod/common/ccd/definition-store-api.yaml

```diff
@@ -28,10 +28,10 @@ spec:
       environment:
         IDAM_USER_URL: "https://hmcts-access.service.gov.uk"
         ELASTIC_SEARCH_FAIL_ON_IMPORT: true
-        DEFINITION_STORE_DB_MAX_POOL_SIZE: 32
+        DEFINITION_STORE_DB_MAX_POOL_SIZE: 25
         DEFINITION_STORE_DB_NAME: ccd_definition_store
         DEFINITION_STORE_DB_HOST: ccd-definition-store-api-postgres-db-v11-prod.postgres.database.azure.com
-        DEFINITION_STORE_TX_TIMEOUT_DEFAULT: 120
+        DEFINITION_STORE_TX_TIMEOUT_DEFAULT: 40
```

Comment:
We have only occasional imports taking 20-35 seconds - 3 of them in the last 7 days.
To tolerate them, I suggested 40 sec here, despite my view for 30 on the call.
Thanks @danlysiak for your question which made me check that aspect further.